README.md

```diff
@@ -22,3 +22,20 @@ to be installed prior to running this.
 Note: this contains `rwc/` a fork of [rust-crypto](https://github.com/DaGenix/rust-crypto)
 without the dependencies that cannot be build easily in a wasm environment, and minus the algorithms
 that is not useful.
+
+Running the Example
+-------------------
+There is a simple example application in `js-example` that can be run to test some of the features.
+
+### installation
+
+within `js-example/` folder
+
+2. `npm i && npm i -g bower`
```

Comment:
It would be better to add bower as an NPM dev dependency instead of installing it globally. Then it can be used from `node_modules` or from the script in `package.json`the same way as if it was installed globally. @DominikGuzei

Reply:
```
"scripts": {
...
    "bower-install": "bower install"
  }
```

`npm run bower-install`

Reply:
Yeah that's right … will update